méli-mélo/2023-09-menu/campaign-menu.js

```diff
@@ -180,18 +126,30 @@
 
                         // Determine if the focus is on a link or an anchor to open a submenu
                         if (savedFocusHref.match("^#")) {
-                            let submenuId = savedFocusHref.substring(6);
-                            $(".gcweb-menu a[aria-controls='sub-menu-" + submenuId + "']").focus();
+                            let submenuId = savedFocusHref.substring(14);
```

Comment:
```
.substring( 14 )
```
That instruction look like to be very chirurgical. Not an issue for this PR, but it will be preferable to calculate the length to be used by the substring instead of using a fix length.